standalone/pruntime/src/api_server.rs

```diff
@@ -388,10 +412,9 @@ pub(super) fn rocket_acl(args: &super::Args) -> Option<rocket::Rocket<impl Phase
         .merge(("port", public_port))
         .merge(("limits", Limits::new().limit("json", 100.mebibytes())));
 
-    let mut server_acl =
-        rocket::custom(figment).mount("/", routes![getinfo, get_contract_info, get_cluster_info]);
+    let mut server_acl = rocket::custom(figment).mount("/", routes![getinfo, help]);
```

Comment:
Why `get_contract_info` and `get_cluster_info` was removed, and `help` was added?

Reply:
I previously added `get_contract_info` and `get_cluster_info` as the JSON version of `/prpc/PhactoryAPI.GetContractInfo` and `/prpc/PhactoryAPI.GetClusterInfo`. Nobody actually uses it so far. Now that the pRPC itself supports JSON now, it is meaningless to keep them there.